src/azure-map/utils/arrayHelp.ts

```diff
@@ -1,16 +1,9 @@
-import {T} from '@angular/core/src/render3';
-
-export interface ArrDiff {
+export interface ArrDiff<T> {
   added: T[];
   deleted: T[];
 }
 
-export interface Foo {
-  name: string;
-  type: string;
-}
-
-export function arrayDiff(old: T[], newArr: T[]): ArrDiff {
+export function arrayDiff<T>(old: T[], newArr: T[]): ArrDiff<T> {
   const difference: ArrDiff = {added: undefined, deleted: undefined};
```

Comment:
What if we do something like 
```javascript
return {
   added: .....
   deleted: .....
}
```
Then we should not have to create object with undefined values.